app/tests/test_multiple_bucket_setup.py

```diff
@@ -0,0 +1,170 @@
+from unittest.mock import patch
```

Comment:
I intentionally put the tests here instead of in one of the subfolder, as they don't really interact with either API.